### PR TITLE
Rename Jalisco to SgxRootEnclave

### PR DESCRIPTION
**Pull request by dominic-mulligan-arm**
Renamed in documentation, comments, source code and build scripts.

### CODE REVIEW

**Comment by dreemkiller** [comment]
It's reporting that `veracruz-utils/src/jalisco_opcodes.rs` has been deleted, and that there's a new file `veracruz-utils/src/sgx_root_enclave_opcodes.rs`. It would be preferable to have the file renamed instead so we can track historic changes.

Did you `git rename` it and the diff just isn't showing it correctly?

File: COMPONENTS.markdown [hunk 1]
```diff
@@ -60,11 +57,11 @@ components:
 - psa-attestation: support code for the Arm PSA Attestation Protocol and
 Token.  This is the attestation protocol that the Veracruz attestation
 service exposes to client code.
-- Sonora: this is the root attestation isolate that is assumed to be present
-on the delegate's machine, and which challenges the authenticity of other
+- SGXRootEnclave: this is the root attestation isolate that is assumed to be present
```
Comment (geky) [inline COMPONENTS.markdown:60]: If we're following Rust's style guide _strictly_ this should be SgxRootEnclave.

I'm still catching up so if we've decided against this feel free to ignore me. And if we're already using "SGX" in the codebase consistency is preferred to style guides.

**Comment by geky** [comment]
@dreemkiller Unfortunately git-rename-diffs are just a heuristic on top of delete A create B, if enough of the file changes git thinks the files are unrelated. This also has the funny effect that sometimes git thinks unrelated files are renames.

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 8b4d72349a7c027a3aa3c624754177a8f7ee99c7
* Result: FAILED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=0df21f2d-e79e-4275-a9a7-148d915c30e1%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by dominic-mulligan-arm** [comment]
> It's reporting that `veracruz-utils/src/jalisco_opcodes.rs` has been deleted, and that there's a new file `veracruz-utils/src/sgx_root_enclave_opcodes.rs`. It would be preferable to have the file renamed instead so we can track historic changes.
> 
> Did you `git rename` it and the diff just isn't showing it correctly?

Yeah, I noticed that before pushing.  It was actually a `git mv` which for some reason Git thinks was delete and then the creation of a new file.

File: COMPONENTS.markdown (hunk 1 above)
Comment (dominic-mulligan-arm) [reply COMPONENTS.markdown:60]: I tried to use `SgxRootEnclave` throughout, but missed this one.  I'll change this.

**Review by dreemkiller — APPROVED** [review]

File: COMPONENTS.markdown (hunk 1 above)
Comment (geky) [reply COMPONENTS.markdown:60]: Ah! That's probably because this is the only one outside of code

**Comment by dominic-mulligan-arm** [comment]
Tests build locally on my machine, so I've marked this ready for review.  Having said that, there's a few changes that are in the merge queue ahead of this, and will therefore require rebasing once they are merged, so it's probably best to delay approval/disapproval for now.

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: cf0ac59c8bc60a0dc8048a6294fbd246db834f74
* Result: FAILED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=b93f93ae-15fe-4a2c-bd71-fa78af3a3d10%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 882fda8ae172764b18c3726d2752fdde2f135910
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=7af67961-5a57-4e61-8a1f-b9796b973cec%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: fc2dc8f3c3bf51b6bdd20fa4cd5c4f5efe2bf334
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=6d5b8079-bc4e-472e-9f7f-26fdda8dbd76%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: d8e8653d4342aa9b11bc090eba9b699ebc118b2b
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=228aa145-94c1-49f3-bc97-f529efc47181%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by dominic-mulligan-arm** [comment]
This is now ready for review.

**Review by ShaleXIONG — APPROVED** [review]

**Comment by dominic-mulligan-arm** [comment]
+1 reached, merging.

**Pull request merged**